### PR TITLE
iterative-telemetry 0.0.9 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
iterative-telemetry 0.0.9 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5393]
- dev_url:        https://github.com/iterative/telemetry-python/tree/0.0.9
- conda_forge:    https://github.com/conda-forge/iterative-telemetry-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/iterative-telemetry/0.0.9
- pypi inspector: https://inspector.pypi.io/project/iterative-telemetry/0.0.9

### Explanation of changes:

- feedstock added to aggregate
- tests require `mocker` which we don't have

[PKG-5393]: https://anaconda.atlassian.net/browse/PKG-5393?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ViridianMelody — COMMENTED** [review]
`mocker` is a pytest fixture provided by `pytest-mock` which we do have on defaults. You should be able to run the tests with just `pytest` and `pytest-mock`

**Comment by psteyer** [comment]
> pytest-mock
> `mocker` is a pytest fixture provided by `pytest-mock` which we do have on defaults. You should be able to run the tests with just `pytest` and `pytest-mock`

Out of curiosity, how were you able to find `pytest-mock`?

**Comment by ViridianMelody** [comment]
> > pytest-mock
> > `mocker` is a pytest fixture provided by `pytest-mock` which we do have on defaults. You should be able to run the tests with just `pytest` and `pytest-mock`
> 
> Out of curiosity, how were you able to find `pytest-mock`?

It was listed in [setup.cfg](https://github.com/iterative/telemetry-python/blob/b6aade50fc39f632ba865e4417d273ab273e3783/setup.cfg#L37)

**Review by ViridianMelody — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,35 +10,46 @@ source:
...
   number: 0
+  skip: true  # [py<38]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools_scm >=6.3.1
+    - wheel
```
Comment (lorepirri) [inline recipe/meta.yaml:22]:
```suggestion
    - wheel
    - setuptools
```

`setuptools` missing

Comment (lorepirri) [reply recipe/meta.yaml:22]: https://github.com/iterative/telemetry-python/blob/0.0.9/pyproject.toml#L2

Comment (psteyer) [reply recipe/meta.yaml:22]: added

**Review by lorepirri — APPROVED** [review]

**Pull request merged**